integration-tests/src/config.rs

```diff
@@ -0,0 +1,69 @@
+use reqwest::Url;
+use serde_json::Value;
+use trustification_event_bus::{EventBusConfig, EventBusType};
+
+use crate::{create_provider, create_provider_context, ProviderContext};
+
+#[derive(Default)]
+pub struct Config {
+    pub spog: Option<Url>,
+    pub bombastic: Option<Url>,
+    pub vexination: Option<Url>,
+    issuer: String,
+    user: String,
+    manager: String,
+    secret: String,
+}
+
+impl Config {
+    pub async fn new() -> Self {
+        let _ = env_logger::try_init();
+        match std::env::var("TRUST_URL") {
```

Comment:
@jcrossley3 This env variable is configured on .env or local?

Reply:
The `.env` file is only used by the compose scripts. `TRUST_URL` needs to be set before running the integs with cargo, e.g.
```
TRUST_URL=http://localhost:8084 cargo test -p integration-tests
```

Reply:
Thanks for the information